contracts/Vesting.sol

```diff
@@ -187,23 +171,20 @@ contract Vesting is IVesting, Ownable {
      * @dev internal function for calculate how many tokens
      * beneficiary can take when call function withdraw
      */
-    function _calculateUnlock(uint256 percentage)
-        internal
-        view
-        returns (uint256)
-    {
-        uint256 onePercentInTokens = (listOfBeneficiaries[msg.sender]
-            .balanceBase / MAX_PERCENTAGE) * percentage;
+    function _calculateUnlock() internal view returns (uint256) {
+        uint256 onePercentInTokens = MAX_UNLOCK_AMOUNT *
+            (listOfBeneficiaries[msg.sender].balanceBase / MAX_PERCENTAGE);
```

Comment:
first should mul then div

Reply:
* then /  = (MAX_UNLOCK_AMOUNT * listOfBeneficiaries[msg.sender].balanceBase)/MAX_PERCENTAGE